### DISCUSSION TITLE
Music Videos category

### DISCUSSION THREAD

**Discussion by Extarys** [Komet/MediaElch#982] [Ideas]
**Is your feature request related to a problem? Please describe.**
I use the "Concert" part to fill my music video details. It takes a long time to fill each song of each artist with as much detail as possible. Just filling the "Artist" field takes a long time.

**Describe the solution you'd like**
Allow "Music" to also scan video files, or;
Create a new "Music Video" section.
Both could use TheAudioDB to fill in the details.

**Describe alternatives you've considered**
I searched a long time for a way to generate nfo files for my music videos. Never found a way to do it. (I'm on Linux)

**Additional context**
I have around 150 music video files that never had a nfo file. I'd like that very much.

**Comment by bugwelle** [comment]
Good idea 👍 
This won't make it to the next version, though (as there are too many other topics I need to take care of).

**Reply by KINGLIFER** [reply]
Has this been looked at as of yet?

**Reply by bugwelle** [reply]
No. :-/
Never got the time.

**Reply by ihiska** [reply]
Также нет времени но я стараюсь

**Reply by ihiska** [reply]
Согласна хорошая идея 👍

**Comment by KINGLIFER** [comment]
Understood. Totally understand.

**Reply by ihiska** [reply]
Понятно но грустно!

**Comment by ihiska** [comment]
Использовать ИИ
